vite.config.ts

```diff
@@ -1,30 +1,7 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
-//@ts-ignore
-import istanbul from 'vite-plugin-istanbul';
-import dts from 'vite-plugin-dts';
-import * as path from 'path';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [
-    react(),
-    dts({
-      insertTypesEntry: true,
-    }),
-    istanbul({
```

Comment:
are we not collecting code coverage anymore?

Reply:
I tried to get rid of `vite` at all, and I was trying to replace it with the `rollup` to have fewer deps, even for development, and when it failed I just miss to return some stuff

Reply:
this is actually a pretty good point though. i was only really using `vite` because of how fast it is in `dev` mode (and also because Library Mode let us have a sandbox app and a `lib` folder that contained the packages to be published), but since we're changing to a monorepo, it might not really be necessary anymore.

probably beyond the scope of this now, but we could probably return to this in the future (`vite` just uses `rollup` for bundling anyway -- its only `esbuild` during dev)